src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball(new Random().nextInt(100), randomColor.getRandomColor());
```

Comment:
`Random` must be a class field
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)